packages/reports/src/Stores/Modules/Portfolio/portfolio-store.js

```diff
@@ -0,0 +1,491 @@
+import throttle from 'lodash.throttle';
```

Comment:
are we going to maintain portfolio-store in both trader and reports package?

Reply:
we need this store in both trader and reports, so I think yes